src/roadmap/releases.md

```diff
@@ -0,0 +1,244 @@
+
+
+## Workflow Releases Every Two Weeks
+
+Deis Workflow has a regular, public release cadence. From v2.0.0 onward, the Deis team releases
+every 2 weeks, with patch versions as needed. GitHub milestones are used to communicate the
+content and timing of major and minor releases.
+
+Workflow release timing is not linked to specific features. If a feature is merged before the
+release date, it is included in the next release.
+
+See "[How to Release Workflow](#how-to-release-workflow)" for more detail.
+
+
+## Semantic Versioning
+
+Deis releases comply with [semantic versioning][semantic version], with the "public API" broadly
+defined as:
+
+- REST, gRPC, or other API that is network-accessible
+- Library or framework API intended for public use
+- "Pluggable" socket-level protocols users can redirect
+- CLI commands and output formats
+
+In general, changes to anything a user might reasonably link to, customize, or integrate with
+should be backward-compatible, or else require a major release. Deis users can be confident that
+upgrading to a patch or to a minor release will not break anything.
+
+
+## How to Release a Component
+
+Most Deis projects are "components" which produce a Docker image or binary executable as a
+deliverable. This section leads a maintainer through creating a component release.
+
+### Step 1: Update Code and Set Environment variables
+
+In the component repository, update from the GitHub remote and ensure `HEAD` is the commit
+intended for release. Major or minor releases should happen on the master branch. Patch releases
+should check out the previous release tag and cherry-pick specific commits from master.
+
+Double-check that `git log` looks correct, then set some environment variables:
+```bash
+export COMPONENT=${PWD##*/}
+export OLD_RELEASE=$(git describe --abbrev=0 --tags)
+export NEW_SHA=$(git rev-parse --short HEAD)
+deisrel changelog individual $COMPONENT $OLD_RELEASE $NEW_SHA unknown
+export NEW_RELEASE=v2.2.1  # changelog agrees it's a patch release
+```
+
+### Step 2: Push the Release Tag
+
+Push the new release tag to the GitHub repository:
+
+```bash
+git tag $NEW_RELEASE && git push upstream $NEW_RELEASE
+```
+
+### Step 3: Put CHANGELOG in GitHub Release Notes
+
+Generate the CHANGELOG with the [`deisrel`](https://github.com/deis/deisrel.git) tool and paste
+it into the body of release notes for the component in GitHub. In the "Release Title" field, use
+the project & component with its release, such as "Deis Controller v2.2.1":
+
+```bash
+deisrel changelog individual $COMPONENT $OLD_RELEASE $NEW_SHA $NEW_RELEASE | pbcopy
+open https://github.com/deis/$COMPONENT/releases/new?tag=$NEW_RELEASE
+```
+
+### Step 4: Publish Release to Versions Server
+
+Finally, use the Workflow Manager API publishing tool to add the new component release to the
+versions server. This step is described in the workflow-manager-api-publish docs and must be done
+by a Deis maintainer.
+
+
+## How to Release Workflow
```

Comment:
Wondering if we should add info as to using `deisrel` to see the `clean/dirty` output of all the components, for informing what tags are supplied in the release charts's `generate_params.toml` in Step 3. below?  See https://github.com/deis/deisrel/pull/122#issuecomment-235944997 for background...